modules/functional-test/testng_integrationTests.xml

```diff
@@ -78,6 +78,11 @@
             <class name="com.intuit.wasabi.tests.service.segmentation.MutualExclusionBatchTest"/>
         </classes>
     </test>
+    <test name="IntegrationStatisticsTests">
+        <classes>
+            <class name="com.intuit.wasabi.tests.service.statistic.StatisticTest"/>
```

Comment:
It is magic :) 
j/k
the testng test class is already committed before we pushed to wasabi